plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -54,6 +55,27 @@ export class GithubPublisher implements PublisherBase {
 
     const { data } = await repoCreationPromise;
 
+    if (access.match(new RegExp(`${owner}/.+`))) {
```

Comment:
Would like some documentation about this functionality

Reply:
what / where?

It's documented in the description on the scaffolder page.

Does it need more than the help text?

Reply:
https://github.com/spotify/backstage/pull/2442/commits/0c7acbe3cbc7f700fede8bad9ac057c08db2a052